public/js/actions/breakpoints.js

```diff
@@ -114,6 +115,25 @@ function removeAllBreakpoints() {
   };
 }
 
+function toggleAllBreakpoints(shouldDisableBreakpoints) {
+  return ({ dispatch, getState }: ThunkArgs) => {
+    const breakpoints = getBreakpoints(getState());
+    return dispatch({
+      type: constants.TOGGLE_BREAKPOINTS,
+      shouldDisableBreakpoints,
+      [PROMISE]: (async function () {
+        for (let [, breakpoint] of breakpoints) {
```

Comment:
do you think this is the best way to destructure the variable?
I chose to use `for of` over `forEach as it seemed simpler with the async